server.js

```diff
@@ -27,9 +19,59 @@ const app = express()
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
+//Model for Book
+const Book = new mongoose.model('Book', {
+  bookID: Number,
+  title: String,
+  authors: String,
+  average_rating: Number,
+  language_code: String,
+  num_pages: Number,
+  ratings_count: Number,
+  text_reviews_count: Number
+})
+
+if (process.env.RESET_DATABASE) {
+  const populateDatabase = async () => {
+    await Book.deleteMany()
+
+    booksData.forEach(item => {
+      const newBook = new Book(item)
+      newBook.save()
+    })
+  }
+  populateDatabase()
+}
+
+//________ Routes/Endpoints ________ //
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.json('👋 Welcome to Emmas book and book reviews API! Use the /books endpoint to see all books 📚🦉. Use the id-parameter to find one book, for example: books/5.')
+})
+
+//Finding all books
+app.get('/books', async (req, res) => {
+  const allBooks = await Book.find()
+
+    if (!allBooks) {
+      res.status(404).json({error: 'Books not found'})
+    } else {
+      res.json(allBooks)
+    } 
+})
+
+//Finding a single book based on bookID
+app.get('/books/:bookID', async (req, res) => {
+  try {
+    const singleBook = await Book.findOne({ bookID: req.params.bookID })
+
+    if (singleBook) {
+      res.json(singleBook)
+    } else {
+      res.status(404).json({error: 'Book not found'})
+    } 
+  } catch (error) {
+      res.status(400).json({error: 'Invalid book id'})
+    }
```

Comment:
Nice job! Can be more useful to go with the isbn instead of id since isbn is the same for the books everywhere